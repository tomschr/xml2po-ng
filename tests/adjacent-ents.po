msgid ""
msgstr ""
"Project-Id-Version: adjacent entities and elements test\n"
"POT-Creation-Date: 2006-04-01 00:02+0200\n"
"PO-Revision-Date: 2006-04-01 01:10+0200\n"
"Last-Translator: Данило\n"
"Language-Team: српски\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: adjacent-ents.xml:8(para)
msgid "&suselinux; &suse-version; Release Notes"
msgstr "&suselinux; &suse-version; Објава издања"

#: adjacent-ents.xml:9(para)
msgid "<application>SUSE Linux</application> &suse-version; is ..."
msgstr "<application>SUSE Линукс</application> &suse-version; је ..."

#: adjacent-ents.xml:10(para)
msgid "&suselinux; <application>&suse-version;</application> is ..."
msgstr "&suselinux; <application>&suse-version;</application> је ..."

#. Put one translator per line, in the form of NAME <EMAIL>, YEAR1, YEAR2
#: adjacent-ents.xml:0(None)
msgid "translator-credits"
msgstr ""
