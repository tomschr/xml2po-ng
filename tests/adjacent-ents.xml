<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE sect1 PUBLIC "-//OASIS//DTD DocBook XML V4.2//EN" "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd" [
<!ENTITY suse "SUSE">
<!ENTITY suselinux "&suse; Linux">
<!ENTITY suse-version "10.1">
]>
<sect1>
 <title><!-- blah --> &suselinux; &suse-version; Release Notes</title>
 <para><application>SUSE Linux</application> &suse-version; is ...</para>
 <para>&suselinux; <application> &suse-version;</application> is ...</para>
</sect1>
