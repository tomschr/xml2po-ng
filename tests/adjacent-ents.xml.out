<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE sect1 PUBLIC "-//OASIS//DTD DocBook XML V4.2//EN" "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd" [
<!ENTITY suse "SUSE">
<!ENTITY suselinux "&suse; Linux">
<!ENTITY suse-version "10.1">
]>
<sect1>
 <title>&suselinux; &suse-version; Објава издања</title>
 <para><application>SUSE Линукс</application> &suse-version; је ...</para>
 <para>&suselinux; <application>&suse-version;</application> је ...</para>
</sect1>
